.github/actions/build_component_wheels/action.yaml

```diff
@@ -0,0 +1,55 @@
+name: 'Build component wheels'
+description: 'Action for building all components wheels'
+inputs:
+  custom_streamlit_component_lib_file:
+    required: false
+    description: 'Streamlit Component Library package'
+    default: ''
+
+outputs:
+  output_directory:
+    description: ''
```

Comment:
Can you add a description here also? Each description field should be filled in.